cmd/common-main.go

```diff
@@ -29,10 +29,12 @@ import (
 
 // Check for updates and print a notification message
 func checkUpdate(mode string) {
-	// Its OK to ignore any errors during getUpdateInfo() here.
-	if older, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
-		if updateMsg := computeUpdateMessage(downloadURL, older); updateMsg != "" {
+	// Its OK to ignore any errors during doUpdate() here.
+	if updateMsg, _, currentReleaseTime, latestReleaseTime, err := getUpdateInfo(1*time.Second, mode); err == nil {
```

Comment:
1s timeout seems too little for non-US clients, for example I get a timeout while `getUpdateInfo` checks for update